AdvancedDLSupport/NativeLibraryBuilder.cs

```diff
@@ -82,6 +82,9 @@ public class NativeLibraryBuilder
         [NotNull]
         private static readonly ConcurrentDictionary<GeneratedImplementationTypeIdentifier, Type> TypeCache;
 
+        private Loaders.ILibraryLoader _customLibLoader;
```

Comment:
Why is the namespace qualification required here? Remove if possible.

Reply:
It was required because it was not originally included in the usings.

Reply:
Just add another using :P